conn_test.go

```diff
@@ -466,15 +484,36 @@ func TestPSK(t *testing.T) {
 	defer report()
 
 	for _, test := range []struct {
-		Name           string
-		ServerIdentity []byte
-		CipherSuites   []CipherSuiteID
+		Name                   string
+		ServerIdentity         []byte
+		CipherSuites           []CipherSuiteID
+		ClientVerifyConnection func(*State) error
+		ServerVerifyConnection func(*State) error
+		WantFail               bool
 	}{
```

Comment:
Maybe adding
```
ExpectedServerErr string
ExpectedClientErr string
```
to each `WantFail=true` test case and checking the expected string is container would be easier to read?

Reply:
Sure. Fixed